receiver/k8sclusterreceiver/internal/clusterresourcequota/metadata.yaml

```diff
@@ -0,0 +1,52 @@
+type: k8s/clusterresourcequota
+
+sem_conv_version: 1.18.0
+
+parent: k8s_cluster
+
+resource_attributes:
+  k8s.namespace.name:
+    description: The k8s namespace name.
+    type: string
+    enabled: true
+
+  openshift.clusterquota.uid:
+    description: The k8s ClusterResourceQuota uid.
+    type: string
+    enabled: true
+
+  openshift.clusterquota.name:
+    description: The k8s ClusterResourceQuota name.
+    type: string
+    enabled: true
+
+  opencensus.resourcetype:
+    description: The OpenCensus resource type.
+    type: string
+    enabled: true
+
+metrics:
+  openshift.clusterquota.limit:
+    enabled: true
+    description: The configured upper limit for a particular resource.
+    unit: "1"
+    gauge:
+      value_type: int
```

Comment:
Please add the following attributes to maintain the current metrics format and regenerate:
- `resource` attribute to `openshift.clusterquota.limit` and `openshift.clusterquota.used` metrics
- `resource` and `k8s.namespace.name` attributes to `openshift.appliedclusterquota.limit` and `openshift.appliedclusterquota.used` metrics
Then you don't need to call `EmitForResource`

Reply:
Thanks for pointing this out! I'll update. I'm planning on testing on an openshift cluster as well before marking this PR ready for to confirm the format hasn't changed. I'll also check to see if the e2e test for this receiver can be updated to test this, but since it's Openshift-specific I'm a bit skeptical.

Reply:
We cannot validate it with the e2e tests. Just use unit tests